packages/styles/scss/components/text-input/_text-input.scss

```
@@ -0,0 +1,384 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Text
+//-----------------------------
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/component-reset';
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/button-reset';
+@use '../form';
+
+/// Text input styles
+/// @access private
+/// @group text-input
+@mixin text-input {
+  .#{$prefix}--text-input {
+    @include reset;
+    @include type-style('body-short-01');
+    @include focus-outline('reset');
+
+    width: 100%;
+    height: rem(40px);
+    padding: 0 $carbon--spacing-05;
```

Spacing tokens don't need the `carbon--` anymore. There's a few more usages in the file that can be updated.

```suggestion
    padding: 0 $spacing-05;
```